keps/sig-node/5004-dra-extended-resource/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: DRA Extended Resource
+kep-number: 5004
+authors:
+  - "@yliaog"
+owning-sig: sig-scheduling
```

Comment:
You'll need to move it to the sig-scheduling directory, too.

Also add the PRR file (I can be the reviewer)

Reply:
moved to sig-scheduling directory